tencentcloud/extension_tke.go

```diff
@@ -42,3 +42,5 @@ const (
 	TKE_ROLE_MASTER_ETCD = "MASTER_ETCD"
 	TKE_ROLE_WORKER      = "WORKER"
 )
+
+var TKE_WORKER_CHARGE_TYPE = []string{CVM_CHARGE_TYPE_PREPAID, CVM_CHARGE_TYPE_POSTPAID}
```

Comment:
master  use TkeCvmCreateInfo get instantce body too. so `TKE_WORKER_CHARGE_TYPE`  can change to `TKE_INSTANCE_CHARGE_TYPE `